indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -163,33 +164,22 @@ public IndexIngestionSpec getIngestionSchema()
   @Override
   public TaskStatus run(final TaskToolbox toolbox) throws Exception
   {
+    final File firehoseTempDir = Files.createTempDir();
```

Comment:
This should be inside the task working dir -- a subdirectory of `toolbox.getTaskWorkDir()` -- not system temp directory.